### PR TITLE
BETA: Register ashokalis.is-a.dev

### PR DESCRIPTION
**Pull request by Ashokali**
Added `ashokalis.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

**Review by triach-rold — CHANGES_REQUESTED** [review]
Provide a URL to your website.

File: domains/ash.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Ashokali",
+        "email": "ashokalisaras@gmail.com"
+    },
+    "record": {
+        "CNAME": "ash.is-a.dev"
```
Comment (CuteDog5695) [inline domains/ash.json:7]: you cant use the domain you are requesting as CNAME

File: domains/ashok.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Ashokali",
+        "email": "ashokalisaras@gmail.com"
+    },
+    "record": {
+        "CNAME": "ashok.is-a.dev"
```
Comment (CuteDog5695) [inline domains/ashok.json:7]: you cant use the domain you are requesting as CNAME

File: domains/ashokalis.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Ashokali",
+        "email": "ashokalisaras@gmail.com"
+    },
+    "record": {
+        "CNAME": "ashokalis.is.a.dev"
```
Comment (CuteDog5695) [inline domains/ashokalis.json:7]: you cant use the domain you are requesting as CNAME

**Review by iostpa — CHANGES_REQUESTED** [review]

**Pull request closed without merging**